src/java/main/org/apache/zookeeper/version/util/VerGen.java

```diff
@@ -149,11 +150,9 @@ public static void main(String[] args) {
                         "Invalid version number format, must be \"x.y.z(-.*)?\"");
                 System.exit(1);
             }
-            int rev;
-            try {
-                rev = Integer.parseInt(args[1]);
-            } catch (NumberFormatException e) {
-                rev = -1;
+            String rev = args[1];
+            if (rev == null || rev.trim().isEmpty()) {
+                rev = "-1";
             }
```

Comment:
Oh, really sorry about that. :disappointed: I squashed the commits from #137, but for whatever reason this latest change didn't make through. :thinking: Well, gonna change now. Thanks for pointing out!